outline

```diff
@@ -115,6 +116,8 @@ Fuse : articles/index.md
 		Sound : api-docs/api/fuse/playsound.json
 
 	Layout : articles/layout/layout.md
+		Responsive Layout : articles/layout/responsive-layout.md
+		Safe Layout : articles/layout/safe-layout.md
```

Comment:
`System.ArgumentException: Duplicate link found in outline on line 118: layout/responsive-layout.html. Previously seen on line 12.`

An article needs to have a single home in the outline; you've added both `layout/responsive-layout.md` and `layout/safe-layout.md` twice.

Reply:
Yeah, in case they're browsing the "Layouts section", they don't have to scroll and search above to find it... I can remove it and only have it displayed under the "Fuse Basics section"